src/main/resources/application.yaml

```diff
@@ -20,3 +25,5 @@ logging:
 management:
   server:
     port: 8081
+
+server.port: 8082
```

Comment:
Maybe we can remove this config and use the default port.

Reply:
Hey @falko, sorry, my bad. If you need to modify it, we also need to modify the configuration in `WorkflowTest.java` 

[WorkflowTest](https://github.com/camunda-community-hub/zeebe-dmn-worker/blob/5775d8def74aa9e6a251fcef7874e4bfd93865d1/src/test/java/io/zeebe/dmn/WorkflowTest.java#L47)

```java
   System.setProperty("zeebe.client.broker.gatewayAddress", gatewayContactPoint);
```